CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
 # OHHTTPStubs — CHANGELOG
+## [4.6.1](https://github.com/AliSoftware/OHHTTPStubs/releases/tag/4.6.1)
```

Comment:
That is a _new_ method, so according to server.org that's gonna make us bump to version 4.7.0